lib/routes/routeBackbeat.js

```diff
@@ -89,7 +95,8 @@ function _checkMultipleBackendRequest(request, log) {
         log.error(errMessage);
         return errors.BadRequest.customizeDescription(errMessage);
     }
-    if ((operation === 'puttagging' || operation === 'deletetagging') &&
+    if (headers['x-scal-storage-type'] === 'aws' &&
```

Comment:
S3 has a constant `versioningNotImplBackends` (in constants.js), you may use the same here, as it's likely to forget updating such piece of code when adding a new backend compared to updating a global constant (you can copy the constant in backbeat constants to avoid doing an arsenal PR but ideally it should be uniquely defined in arsenal).

Reply:
Good idea. Updated.